Gutenberg/cocoapods_helpers.rb

```diff
@@ -68,23 +68,25 @@ def gutenberg_pod(config: GUTENBERG_CONFIG)
     options = { path: File.exist?(local_gutenberg) ? local_gutenberg : DEFAULT_GUTENBERG_LOCATION }
 
     raise "Could not find Gutenberg pod at #{options[:path]}. You can configure the path using the #{local_gutenberg_key} environment variable." unless File.exist?(options[:path])
-  else
-    options[:git] = "https://github.com/#{GITHUB_ORG}/#{REPO_NAME}.git"
-    options[:submodules] = true
-  end
 
-  pod 'Gutenberg', options
-  pod 'RNTAztecView', options
+    pod 'Gutenberg', options
+    pod 'RNTAztecView', options
 
-  gutenberg_dependencies(options: options)
+    gutenberg_dependencies(options: options)
+  elsif options[:tag]
+    pod 'Gutenberg', podspec: "https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{options[:tag]}.podspec"
+  elsif options[:commit]
+    pod 'Gutenberg', podspec: "https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{options[:commit]}.podspec"
+  end
```

Comment:
What do you think merging the two if blocks into one?
```suggestion
  else
    pod 'Gutenberg', podspec: "https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-#{options[:tag] || options[:commit]}.podspec"
  end
```

Reply:
Good point. Built on top of this suggestion in 1d8deb1b6cce8a5faa4af976d42506b454147d3f to also add an error if neither is set.